metrics/src/db.py

```diff
@@ -0,0 +1,121 @@
+#   -*- coding: utf-8 -*-
+#
+#   This file is part of portal-metrics
+#
+#   Copyright (C) 2024 SKALE Labs
+#
+#   This program is free software: you can redistribute it and/or modify
+#   it under the terms of the GNU Affero General Public License as published by
+#   the Free Software Foundation, either version 3 of the License, or
+#   (at your option) any later version.
+#
+#   This program is distributed in the hope that it will be useful,
+#   but WITHOUT ANY WARRANTY; without even the implied warranty of
+#   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#   GNU Affero General Public License for more details.
+#
+#   You should have received a copy of the GNU Affero General Public License
+#   along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import logging
+from datetime import date, timedelta
+from typing import List, Dict, Any
+from decimal import Decimal
+
+from peewee import fn, IntegrityError
+
+from src.models import db, Address, TransactionCount
+from src.config import NETWORK_NAME
+from src.explorer import get_address_counters_url
+
+logger = logging.getLogger(__name__)
+
+TRANSACTION_COUNT_FIELD = 'transactions_count'
+BACKFILL_DB_DAYS = 30
```

Comment:
Probably it's better to move db constants to src/config.py

Reply:
moved